examples/bert/README.md

```diff
@@ -5,47 +5,6 @@ Representations from Transformers (BERT) model end-to-end using the KerasNLP
 library. This README contains instructions on how to run pretraining directly
 from raw data, followed by finetuning and evaluation on the GLUE dataset.
 
-## Quickly test out the code
-
-To exercise the code in this directory by training a tiny BERT model, you can
-run the following commands from the base directory of the repository. This can
```

Comment:
I still find this script helpful. 

Even though we have the same code in `bert_test.py`, code in README is more clear for reading and trying.